lib/gui/app/modules/image-writer.js

```diff
@@ -392,7 +392,9 @@ exports.cancel = () => {
 
   try {
     const [ socket ] = ipc.server.sockets
-    ipc.server.emit(socket, 'cancel')
+    if (socket !== undefined) {
```

Comment:
Should we add ...

```
// eslint-disable-next-line no-undefined
```

... to make linter happy? Same "comment" is in other parts of the source code.

Reply:
updated